### PR TITLE
7 - Storing settings in the config file

### PR DESCRIPTION
**Pull request by v-ilin**
Issue #7

### CODE REVIEW

File: code/FsComposite/Composite.Test/App.config [hunk 1]
```diff
@@ -1,6 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
+
 <configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7" />
-    </startup>
+  <appSettings>
+    <add key="GithubConfigPath" value="..\..\DataSourceConfigs\github_config.json" />
```
Comment (vkamiansky) [inline code/FsComposite/Composite.Test/App.config:5]: this one is interesting. where does it come from?

**Review by vkamiansky — COMMENTED** [review]
One config line is funny. Otherwise ok.

File: code/FsComposite/Composite.Test/App.config (hunk 1 above)
Comment (v-ilin) [reply code/FsComposite/Composite.Test/App.config:5]: In the ProcessSample project there is same setting in the app.config file.

**Review by vkamiansky — APPROVED** [review]
This one seems ok.

**Pull request merged**